ports/net/openssh/Portfile

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+default_variants +gsskex
```

Comment:
Does this get added to the existing default variants (kerberos5 and xauth) or does it replace them?